src/renderer.cpp

```diff
@@ -742,6 +744,38 @@ void Renderer::prepareShadowMap()
     );
 }
 
+void Renderer::prepareSpecularIbl(char* fullpath)
+{
+    if (bgfx::isValid(specularIblTexture))
+        bgfx::destroy(specularIblTexture);
+
+    uint32_t width, height, mipCount = 0;
+    specularIblTexture = createTextureHandle(fullpath, &width, &height, &mipCount, false);
+    lighting.setIblMipCount(mipCount);
+}
+
+void Renderer::prepareDiffuseIbl(char* fullpath)
+{
+    if (bgfx::isValid(diffuseIblTexture))
+        bgfx::destroy(diffuseIblTexture);
+
+    uint32_t width, height, mipCount = 0;
+    diffuseIblTexture = createTextureHandle(fullpath, &width, &height, &mipCount, false);
+}
+
+void Renderer::prepareEnvBrdf()
+{
+    static char fullpath[MAX_PATH];
+
+    if (bgfx::isValid(envBrdfTexture))
+        bgfx::destroy(envBrdfTexture);
+
+    sprintf(fullpath, "%s/%s/%s.%s", basedir, external_ibl_path.c_str(), "envBrdf", "dds");

```

Comment:
If "envBrdf" and "dds" is a known constant, just put it in the string already, no need to pass them as an argument :)

Reply:
I merged it into "envBrdf.dds".